package.json

```diff
@@ -120,10 +123,12 @@
     "test-js": "wp-scripts test-unit-js",
     "test-php": "./vendor/bin/phpunit",
     "test-php-coverage": "php -dxdebug.mode=coverage ./vendor/bin/phpunit --coverage-html=coverage",
-    "e2e-docker:up": "npm explore @woocommerce/e2e-environment -- npm run docker:up",
-    "e2e-docker:down": "npm explore @woocommerce/e2e-environment -- npm run docker:down",
-    "test:e2e": "npm explore @woocommerce/e2e-environment -- npm run test:e2e",
-    "test:e2e-dev": "npm explore @woocommerce/e2e-environment -- npm run test:e2e-dev",
+    "legacy:e2e-docker:up": "npm explore @woocommerce/e2e-environment -- npm run docker:up",
+    "legacy:e2e-docker:down": "npm explore @woocommerce/e2e-environment -- npm run docker:down",
+    "legacy:test:e2e": "npm explore @woocommerce/e2e-environment -- npm run test:e2e",
```

Comment:
Should we keep these scripts? I think these legacy e2e-tests don't work at all.

Reply:
@merkushin My plan is to remove them as soon we cover the wizard setup scenario, that is the next priority.